docs/guides/computed-properties.mdx

```diff
@@ -6,23 +6,39 @@ description: 'Use object getters and setters'
 
 # Computed Properties
 
-Valtio itself doesn't provide a functionality for computed properties,
-but you can emulate it with object getters and setters.
+In Valtio you can use object & class getters and setters to create computed properties.
 
-⚠️ Note: This is a very advanced usage.
+⚠️ Note: Getters in JavaScript are a more advanced feature of the language, so Valtio recommends using them with caution. That said, if you are a more advanced JavaScript programmer, they should work as you expect; see the "Note about this `this`" section below.
 
-## Simple getter
+## Simple object getter
 
 ```js
 const state = proxy({
   count: 1,
   get doubled() {
+    // We recommend `state.count` for clarity to beginners, but `this.count` works too
```

Comment:
@dai-shi disclaimer I still kept a `this.count` in a comment :see_no_evil: but will remove if you call me on it. :-)

Reply:
I'm fine with this. Thanks.